project/Dependencies.scala

```diff
@@ -20,8 +20,8 @@ object Dependencies {
   val awsCloudWatch = "com.amazonaws" % "aws-java-sdk-cloudwatch" % awsClientVersion
   val scalaz = "org.scalaz" %% "scalaz-core" % "7.3.8"
   val anorm = "org.playframework.anorm" %% "anorm" % "2.7.0"
-  val netty = "io.netty" % "netty-codec" % "4.1.87.Final"
-  val nettyHttp = "io.netty" % "netty-codec-http" % "4.1.87.Final"
+  val netty = "io.netty" % "netty-codec" % "4.1.118.Final"
+  val nettyHttp = "io.netty" % "netty-codec-http" % "4.1.118.Final"
```

Comment:
not worth revisiting now, but for future, if there are two packages that have to be kept in step, it can be extracted e.g.
```scala
val nettyVersion = "4.1.118.Final"
val netty = "io.netty" % "netty-codec" % nettyVersion
val nettyHttp = "io.netty" % "netty-codec-http" % nettyVersion
```